packages/foam-vscode/web/webpack.config.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+const webpack = require('webpack');
+
+/** @typedef {import('webpack').Configuration} WebpackConfig **/
+/** @type WebpackConfig */
+const webExtensionConfig = {
+  mode: 'none', // this leaves the source code as close as possible to the original (when packaging we set this to 'production')
+  target: 'webworker', // extensions run in a webworker context
+  entry: {
+    extension: './src/extension.ts', // source of the web extension main file,
+    'test/web/index': './src/test/web/index.ts',
+  },
+  output: {
+    clean: true,
+    filename: '[name].js',
+    path: path.join(__dirname, './../out/web'),
+    libraryTarget: 'commonjs',
+    devtoolModuleFilenameTemplate: '../../../[resource-path]',
+  },
+  resolve: {
+    mainFields: ['browser', 'module', 'main'],
+    extensions: ['.ts', '.js'], // support ts-files and js-files
+    alias: {
+      // Overwrite wikilink embedding as it will not work in the current setup on web
```

Comment:
2 comments:
1. I wonder if we can handle it from within `wikilink-embed.ts` so that it's more explicit (I don't love webpack nor its black magic ;) )
2. if approach n.1 is not good, I would actually have `/src/web/features/preview/wikilink-embed.ts` -> `src/features/preview/wikilink-embed-for-web-extension.ts` (and add a comment on both files to explain what's happening)

Reply:
First won't be possible. As the file contains an import of `fs` it will try to pack that. By overwriting the file on compile time, I am preventing these issues.